pythonie/meetups/models.py

```diff
@@ -57,4 +58,5 @@ def __str__(self):
     @classmethod
     def future_events(cls):
         today = datetime.now()
-        return cls.objects.filter(time__gt=today).filter(time__lt=next_n_months(today, 3))
+        return cls.objects.filter(
+            time__gt=today).filter(time__lt=next_n_months(today, 3))
```

Comment:
Hi @diarmuidbourke , @nicolasunravel , okay I will not change anything here in this commit (beyond formatting) in case I break the filter.